plugins/inputs/exec/exec.go

```diff
@@ -30,7 +30,10 @@ const sampleConfig = `
   timeout = "5s"
 
   ## measurement name suffix (for separating different commands)
-  name_suffix = "_mycollector"
+	name_suffix = "_mycollector"
```

Comment:
There seems to be a formatting problem...

Reply:
```suggestion
  name_suffix = "_mycollector"
```